pkgs/applications/audio/cardinal/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  stdenv
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, fetchzip
+, freetype
+, jansson
+, lib
+, libGL
+, libX11
+, libXcursor
+, libXext
+, libXrandr
+, libarchive
+, liblo
+, libsamplerate
+, mesa
+, pkg-config
+, python3
+, speexdsp
+}:
+
+stdenv.mkDerivation rec {
+  name = "cardinal-${version}";
+  version = "22.02";
+
+  src = fetchurl {
```

Comment:
Why not use `fetchFromGitHub`?

Reply:
When I use this:
```nix
  src = fetchFromGitHub {
    owner = "DISTRHO";
    repo = "Cardinal";
    rev = version;
    sha256 = "sha256-mg+BzXIciPs+6pDdjBAHsoP3w4cyK/+KA2MyTwgTVX8=";
  };
```
I get the wrong file.